JULIERAJ/KIDS-FIRST-Back
# refactor(models): removing unused models

Description
Remove unnecessary models from the backend[#23]

Requirements:
There are some models that are no longer relevant to our current database schema and should be removed such as invitation, member, family.

Test Steps
1. Setup Postman:
Open Postman and create a new collection for your API tests.

Testing Endpoints:

Get All Kids:

Method: GET
URL: http://localhost:8000/api/kids
Headers: None
Body: None

Expected Response: A success message with a status of 200.
Create Kid:

Method: POST
URL: http://localhost:8000/api/kids
Headers: Content-Type: application/json
Body (example):
json
{
  "name": "John Doe",
  "age": 10,
  "parent_id": "parent-id-123"
}
Expected Response: The created kid profile in JSON format with a status of 201.

Get Kid by ID:

Method: GET
URL: http://localhost:8000/api/kids/kjkjk
Headers: None
Body: None
Expected Response: A success message with a status of 200.
Update Kid:

Method: PUT
URL: http://localhost:8000/api/kids/kjkjk
Headers: Content-Type: application/json
Body (example):None
Expected Response: A success message with a status of 200.

Delete Kid:

Method: DELETE
URL: http://localhost:8000/api/kids/hjhjjh
Headers: None
Body: None
Expected Response: A success message with a status of 200.

API Changes
New Endpoints: None
Removed Endpoints: None
Modified Endpoints: None


Checklist
 Pull Request title includes the issue number and a brief description of the change.
 All changes should be tested and verified to work correctly.
 Code follows backend best practices.
 Branch names follow the conventions in the contributing file.
 Commit messages follow the naming conventions in the contributing file.
 No sensitive data or secrets are included in the codebase.
